File: ppldispatchapp1/ppldispatchapp1/ClientApp/src/app/models/CaseSearch.model.ts
export class CaseSearch {
    constructor(beginDate: string, EndDate?: string, Employee?: number, Customer?: number) {
        this.BeginDate = beginDate;
        this.EndDate = EndDate;
        this.Employee = Employee;
        this.Customer = Customer;

    }

    public BeginDate: string;
    public EndDate: string;
    public Employee: number;
    public Customer: number;
}

File: ppldispatchapp1/DAL/Models/CaseSearchModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models
{
    public partial class CaseSearchModel 
    {
        
        public string BeginDate { get; set; }
        public int Customer { get; set; }
        public int Employee { get; set; }
        public string EndDate { get; set; }
        public string ApartmentNumber { get; set; }
    }
}
